### PR TITLE
made length of descriptions and number of timings configurable in bac…

### PR DESCRIPTION
**Pull request by CamillH**
…kend

### CODE REVIEW

File: ext_conf_template.txt [hunk 1]
```diff
@@ -6,3 +6,9 @@ sentry_sample_rate =
 
 # cat=sentry; type=string; label=Sentry CLI Sample Rate between 0.0 and 1.0 (empty: keep default)
 sentry_cli_sample_rate =
+
+# cat=sentry; type=integer; label=Number of timings (reduce to make header output smaller. Try out when you get regularly 502 responses).
+number_of_timings = 70
```
Comment (Kanti) [inline ext_conf_template.txt:11]: maybe set this to 30?

File: ext_conf_template.txt [hunk 2]
```diff
@@ -6,3 +6,9 @@ sentry_sample_rate =
...
+
+# cat=sentry; type=integer; label=Length of description (reduce to make header output smaller. Try out when you get regularly 502 responses).
+length_of_description = 100
```
Comment (Kanti) [inline ext_conf_template.txt:14]: maybe set this to 60

File: Classes/Utility/TimingUtility.php [hunk 3]
```diff
@@ -138,12 +138,20 @@ public function shutdown(ScriptResult $result): ?ResponseInterface
...
+            $durations[] = $stopwatch->getDuration();
         }
 
-        if (count($timings) > 70) {
-            $timings = [$this->timingString(0, 'To Many measurements ' . count($timings), 0.000001)];
+        rsort($durations);
+
+        foreach ($stopWatches as $index => $time) {
+            $duration = $time->getDuration();
+            if ($duration >= ($durations[$this->configService->getMaxNumberOfTimings() - 1] ?? 0)) {
```
Comment (Kanti) [inline Classes/Utility/TimingUtility.php:152]: `($durations[$this->configService->getMaxNumberOfTimings() - 1] ?? 0)` should be a variable outside the loop

**Review by Kanti — CHANGES_REQUESTED** [review]

File: ext_conf_template.txt (hunk 1 above)
Comment (CamillH) [reply ext_conf_template.txt:11]: reasonable

**Review by Kanti — APPROVED** [review]

**Pull request merged**